project/apps/publications/src/app/publication/publication.entity.ts

```diff
@@ -29,8 +24,8 @@ export class PublicationEntity<Publication extends BasePublication = BasePublica
   }
 
   protected populate(data: Publication) {
-    this.id = data.id ?? undefined
-    this.tags = data.tags?.map(tag => PublicationTagEntity.fromObject(tag)) ?? undefined
+    this.id = data.id
+    this.tags = data.tags?.map(tag => PublicationTagEntity.fromObject(tag))
```

Comment:
this.tags = (data.tags || []).map(tag => PublicationTagEntity.fromObject(tag))
Comment

this.tags = data.tags?.map(tag => PublicationTagEntity.fromObject(tag))
Comment || []